src/ApolloProvider.tsx

```diff
@@ -49,6 +50,10 @@ export default class ApolloProvider<TCache> extends Component<ApolloProviderProp
   }
 
   render() {
-    return this.props.children;
+    return (
+      <ApolloContext.Provider value={this.getChildContext()}>
```

Comment:
Should also remove `static childContextTypes` from this component.

Reply:
@joshjg I left this intentionally. This PR isn't intended to be a full blown replacement of the old context handling approach, as it's still being used by React Apollo internally. There are more changes required to be able to remove `childContextTypes` completely (which will be coming soon, in `react-apollo` 3.0).